app/services/service_listeners/publishing_api_pusher.rb

```diff
@@ -35,15 +35,30 @@ def push(event:, options: {})
         api.discard_draft_async(edition)
       end
 
+      handle_corporate_information_pages(event)
       handle_html_attachments(event)
+      handle_publications(event)
     end
 
   private
 
+    def handle_corporate_information_pages(event)
```

Comment:
Would it be more explicit to call this (and the publication one) `republish_` rather than `handle_` as that is they do?